lib/ios/RNNBackButtonOptions.m

```diff
@@ -4,46 +4,41 @@
 @implementation RNNBackButtonOptions
 
 - (void)applyOn:(UIViewController *)viewController {
-	if (self.showTitle && ![self.showTitle boolValue]) {
-		self.title = @"";
+  	UIBarButtonItem *backItem = [[UIBarButtonItem alloc] init];
+  
+	backItem.image = self.tintedIconIfAvailable;
```

Comment:
The system back button should be hidden when you are using an icon. 
We need to add this code in case `self.tintedIconIfAvailable` return value
```
[viewController.navigationController.navigationBar setBackIndicatorImage:[UIImage new]];	
[viewController.navigationController.navigationBar setBackIndicatorTransitionMaskImage:[UIImage new]];
```
Thanks! great PR